### DISCUSSION TITLE
java-enhanced-switch/

### DISCUSSION THREAD

**Discussion by giscus** [vojtechruz/vojtechruzicka-blog#71] [Giscus Comments]
# java-enhanced-switch/

Java 12 introduced a whole lot of useful improvements to the good old switch, which makes it way more useful.

https://www.vojtechruzicka.com/java-enhanced-switch/

**Comment by Stive889** [comment]
excellent article, thx

**Reply by vojtechruz** [reply]
Thanks, glad you found it useful :)
